src/classesToPurge.js

```diff
@@ -0,0 +1,17 @@
+let classes;
+async function getClassesToPurge() {
+  if (classes) return classes;
+  else {
+    classes = await fetch(
+      'https://assets.finn.no/pkg/@warp-ds/css/2.0.0-v2.1/classesToPurge.json',
```

Comment:
To be changed to v2 before merge
```suggestion
      'https://assets.finn.no/pkg/@warp-ds/css/v2/classesToPurge.json',
```